src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs

```diff
@@ -653,6 +658,38 @@ private CodeGenOutput GenerateFastifyPluginOutput(ServiceInfo service)
 						{
 							code.WriteLine($"url: '{fastifyPath}',");
 							code.WriteLine($"method: '{httpMethodInfo.Method.ToUpperInvariant()}',");
+							using (code.Block("schema: {", "},"))
+							{
+								using (code.Block("response: {", "},"))
+								{
+									foreach (var response in httpMethodInfo.ValidResponses)
+									{
+										var statusCode = (int) response.StatusCode;
+										code.Write($"{statusCode}: ");
+
+										if (response.BodyField is not null)
+										{
+											code.WriteLine($"{GetJsonSchemaType(response.BodyField.ServiceField.TypeName)},");
```

Comment:
```suggestion
											code.WriteLine($"{GetJsonSchemaType(service.GetFieldType(response.BodyField.ServiceField))},");
```
This would be a more typical implementation, which would allow `GetJsonSchemaType` to switch on `fieldType.Kind` instead of reverse-engineering the type name.

Reply:
You beat me to it! I was going to ask if there was a better way, which I assumed there was. 👍